test/python/result/test_counts.py

```diff
@@ -252,3 +252,43 @@ def test_hex_outcomes_0b_bitstring_counts(self):
         counts_obj = counts.Counts(raw_counts)
         result = counts_obj.hex_outcomes()
         self.assertEqual(expected, result)
+
+    def test_empty_bitstring_counts(self):
+        raw_counts = {}
+        expected = {}
+        result = counts.Counts(raw_counts)
+        self.assertEqual(expected, result)
+
+    def test_empty_bistring_counts_with_exta_formatting_data(self):
+        raw_counts = {}
+        expected = {}
+        result = counts.Counts(raw_counts, 'test_counts',
+                               creg_sizes=[['c0', 2], ['c0', 1], ['c1', 1]],
+                               memory_slots=4)
+        self.assertEqual(result, expected)
+
+    def test_int_outcomes_with_empty_counts(self):
+        raw_counts = {}
+        expected = {}
+        counts_obj = counts.Counts(raw_counts)
+        result = counts_obj.int_outcomes()
+        self.assertEqual(expected, result)
+
+    def test_most_frequent_empty_bitstring_counts(self):
+        raw_counts = {}
+        counts_obj = counts.Counts(raw_counts)
+        self.assertRaises(exceptions.QiskitError,
+                          counts_obj.most_frequent)
+
+    def test_most_frequent_duplicate_empty_bitstring_counts(self):
+        raw_counts = {}
+        counts_obj = counts.Counts(raw_counts)
+        self.assertRaises(exceptions.QiskitError,
+                          counts_obj.most_frequent)
```

Comment:
Should this test be duplicated?

Reply:
oops yeah, sloppy copy paste from the other input formats. This was the test case where counts where like `{0: 10, 1: 10}` and checking that properly raises. But when the raw counts are empty that doesn't mean anything. I'll remove this real quickly

Reply:
Fixed in: https://github.com/Qiskit/qiskit-terra/pull/5018/commits/22d517f4c22c1e159884b105db5b14d4a860d9dd